idflakies-legacy/src/main/java/edu/illinois/cs/dt/tools/detection/LegacyDetectorPathManager.java

```diff
@@ -0,0 +1,128 @@
+package edu.illinois.cs.dt.tools.detection;
+
+import com.google.common.base.Preconditions;
+
+import edu.illinois.cs.dt.tools.utility.Level;
+import edu.illinois.cs.dt.tools.utility.Logger;
+import edu.illinois.cs.dt.tools.utility.PathManager;
+
+import edu.illinois.cs.testrunner.configuration.Configuration;
+import edu.illinois.cs.testrunner.coreplugin.TestPluginUtil;
+import edu.illinois.cs.testrunner.util.ProjectWrapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class LegacyDetectorPathManager extends PathManager {
+    
+    @Override
+    public Path detectionResultsInstance() {
+        return path(DETECTION_RESULTS);
+    }
+
+    @Override
+    public Path detectionFileInstance() {
+        return detectionResultsInstance().resolve(FLAKY_LIST_PATH);
+    }
+
+    @Override
+    public Path pathWithRoundInstance(final Path path, final String testName, final int round) {
+        if (testName == null || testName.isEmpty()) {
+            return path.resolve("round" + String.valueOf(round) + ".json");
+        } else {
+            return path.resolve(testName + "-round" + String.valueOf(round) + ".json");
+        }
+    }
+
+    @Override
+    public Path detectionRoundPathInstance(final String name, final int round) {
+        return pathWithRoundInstance(detectionResultsInstance().resolve(name), "", round);
+    }
+
+    @Override
+    public Path filterPathInstance(final String detectorType, final String filterType, final int absoluteRound) {
+        return detectionRoundPathInstance(detectorType + "-" + filterType, absoluteRound);
+    }
+
+    @Override
+    public Path originalOrderPathInstance() {
+        return path(ORIGINAL_ORDER);
```

Comment:
I would like it to be more explicit about where these constants are coming from, let's add the class name to it, e.g., PathManager.ORIGINAL_ORDER.